server/controllers/incidents.js

```diff
@@ -10,44 +10,58 @@ module.exports = {
   create(req, res) {
     let locationPromise;
     name = req.body.name,
-    centre = req.body.centre,
-    country = req.body.country
+      centre = req.body.centre,
```

Comment:
Remove unnecessary indentation. Align with `name`.

Reply:
@Mbarak-Mbigo on the backend, we are using camel-case conventions for every column  but when referencing models in sequelize, we cannot use small case. We are reusing the models in different parts of the files. The nested objects however are in the expected conventions. Tried using aliases, doesn't work yet. cc: @kimobrian